feature/app.js

```diff
@@ -6,7 +6,7 @@ const logger = require('morgan');
 const mongoose = require('mongoose');
 
 const indexRouter = require('./routes/index');
-// const usersRouter = require('./routes/users');
```

Comment:
Unwanted comments should not be present in PR.

Reply:
Has been removed in the last commit.